docker-compose.yml

```diff
@@ -0,0 +1,187 @@
+version: "3.4"
+
+services:
+  redis:
+    image: redis:3.2.11
+    ports:
+      - 6379:6379
+    networks:
+      - spectrum
+
+  rethinkdb:
```

Comment:
Why add a service subdirectory, I think it's enough to use /.

Reply:
Because I also added yarn workspaces and this is a preparation step for cleaning up the node modules dependency mess across the board. It's also helpful for keeping the docker context small without maintaining a massive dockerignore file.